### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by dlfivefifty**

### CODE REVIEW

File: labs/lab2.ipynb [hunk 1]
```diff
@@ -0,0 +1,960 @@
...
+   ],
+   "metadata": {},
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Problem 4(b)** Use dual numbers to compute the derivatives to\n",
+    "$$\n",
+    "\\exp(\\exp x \\cos x + \\sin x), ∏_{k=1}^{1000} \\left({x \\over k}-1\\right), \\hbox{ and } f^{\\rm s}_{1000}(x).\n",
```
Comment (SamBrzezicki) [inline labs/lab2.ipynb]: maybe space out the three functions a little

File: labs/lab2.ipynb [hunk 2]
```diff
@@ -0,0 +1,960 @@
...
+    "------\n",
+    "## Newton's method"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Newton's method is a simple algorithmic approach for computing roots (or zeros)\n",
+    "of functions that you may have seen before in A-levels. The basic idea is given an initial guess $x_0$,\n",
```
Comment (SamBrzezicki) [inline labs/lab2.ipynb]: I'd change A-levels to school (as they come from many backgrounds)

File: labs/lab2.ipynb [hunk 3]
```diff
@@ -0,0 +1,960 @@
...
+    "find the first-order Taylor approximation $p(x)$ (i.e., fine the line that matches the slope of the function at the point)\n",
```
Comment (SamBrzezicki) [inline labs/lab2.ipynb]: find (typo)

File: src/labs/lab2s.jl [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+## END
+
+
+
+# ------
+# ## Newton's method
+
+# Newton's method is a simple algorithmic approach for computing roots (or zeros)
+# of functions that you may have seen before in A-levels. The basic idea is given an initial guess $x_0$,
+# find the first-order Taylor approximation $p(x)$ (i.e., fine the line that matches the slope of the function at the point)
```
Comment (dlfivefifty) [inline src/labs/lab2s.jl]:
```suggestion
# find the first-order Taylor approximation $p(x)$ (i.e., find the line that matches the slope of the function at the point)
```

**Pull request merged**